core/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/runtimeModels.scala

```diff
@@ -511,9 +511,12 @@ object MemorySize {
  */
 final case class RuntimeResourceConstraints(memoryLimit: MemorySize)
 
-final case class RunningRuntime(cloudContext: CloudContext,
```

Comment:
`RunningRuntime` was previously used by `ClusterToolMonitor` (which I deleted) -- renamed to `RuntimeMetrics`